.gitignore

```diff
@@ -27,9 +27,14 @@ sphinx/source/_build
 # Py.test
 /.cache/
 
-# Eclipse, PyDev
+# Eclipse, PyDev, VS Code
 .project
 .pydevproject
+.vscode
+
+# virtualenv
+VENV
```

Comment:
Modern env managers allow keeping env folders outside of the repo. I suggest you using those, like pyenv, for example. Who knows whether we decide having these folder names in repo in future?

/cc: @jaraco

Reply:
Ah, Ok, Well sorry to bother you with my outdated tools.

Reply:
No worries. I could still accept vscode part. I didn't, because it wasn't a separate PR.

Reply:
I've switched to using the global .gitignore. I was not aware of that feature. 

We use virtualenv because it is built-in to python. We use built-in libraries (batteries included) in all cases without a very very good reason to use an external library.

Reply:
Oh, and using this approach you're not restricted to use some conventional `venv/` dir for environment anymore to match someone else's ignoring rules. You could use your own naming conventions: I, for example, include project name and python interpreter version into env name even with `pyenv` tool (which btw is able to automatically activate env once I enter project folder).